state-chain/pallets/cf-validator/src/lib.rs

```diff
@@ -145,6 +142,9 @@ pub mod pallet {
 		/// This is used to reset the validator's reputation
 		type ReputationResetter: ReputationResetter<ValidatorId = ValidatorIdOf<Self>>;
 
+		/// The Flip token implementation.
+		type Flip: AccountInfo<Self>;
```

Comment:
The Flip pallet is already linked via Chainflip::FundingInfo so we don't really need to add it here. 

We could just add the AccountInfo constraint to the Chainflip trait:

```rust
pub trait Chainflip {
	// ...
	type FundingInfo:
		FundingInfo<AccountId = Self::AccountId, Balance = Self::Amount> +
		AccountInfo<Self>;
}

Reply:
Added this to the mock. Also added AccountInfo to the MockFundInfo